profile.yaml

```diff
@@ -7,5 +7,8 @@ spec:
   version: v0.0.1
   artifacts:
     - name: nginx-server
+      kind: HelmChartRemote
       path: nginx/chart
-      kind: HelmChart
+      helm_url: https://charts.bitnami.com/bitnami
+      helm_chart: nginx
+      helm_chart_version: "8.8.1"
```

Comment:
this looks like its both a `HelmChartLocal` and a `HelmChartRemote`? Is this supposed to be two separate artifacts?

Reply:
It was an attempt to somehow make the test work. But unfortunately, Jake, I think we are going to need to do something with multiple artifacts which is in your PR to get this working properly. The tests I mean.